biopet-utils/src/main/scala/nl/lumc/sasc/biopet/utils/implicits/Either.scala

```diff
@@ -0,0 +1,40 @@
+package nl.lumc.sasc.biopet.utils.implicits
+
+/**
+ * Created by pjvanthof on 20/02/2017.
+ */
+object Either {
```

Comment:
Did make a issue in this epic for to look into this. For this merge is can be ignored, will be in the next batch.